bpf/node_config.h

```diff
@@ -49,6 +49,13 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 
 #define HOST_ID 1
 #define WORLD_ID 2
```

Comment:
We need to have a broader conversation about this with datapath folks. I would categorize this as a possible next step, but it changes the UX of the policy engine. We should do that in another PR.

Reply:
Why does it change the UX of policy engine? It seems like this PR already splits world into three identities depending on L3 protocol, so if this has any repercussions then I think that's important to raise & engage with, since folks will need to develop future datapath changes based on that understanding.

(I didn't see anything particularly concerning in this PR and I don't think that this needs to block merging this bugfix, but I do want to make sure this discussion happens with datapath folks).

Reply:
> Why does it change the UX of policy engine?

Well, maybe not directly, but these entities could be selectable. Of course we could figure out a way to not expose them as referenceable entities in a policy file.

Reply:
I don't think they need to be user-facing entities. It's sufficient for these _identities_ selected by the existing policy APIs, as an internal detail.

Reply:
^ No reason to block this PR on this discussion point, but we can continue to discuss out-of-band on the implications here and longer term supportability.